apps/design/backend/src/worker/generate_election_package_and_ballots.ts

```diff
@@ -144,11 +147,19 @@ export async function generateElectionPackageAndBallots(
     JSON.stringify(metadata, null, 2)
   );
 
+  const userDefinedHmpbStrings = getUserDefinedHmpbStrings(precincts);
+  // Combine predefined and user-defined HMPB strings here because they can be
+  // translated in the same path.
+  const combinedHmpbStringsCatalog: Record<string, string> = {
+    ...hmpbStringsCatalog,
+    ...userDefinedHmpbStrings,
+  };
```

Comment:
Should this logic be part of `getAllStringsForElectionPackage` so it's executed at other call sites as well? (I think currently there's just one in the fixture-generators library)